serial/serbridge.c

```diff
@@ -13,6 +40,15 @@
 
 #define SKIP_AT_RESET
 
+
+/*  ESP8266
+ */
+#define IROM			ICACHE_FLASH_ATTR
+#define REG_BRR			(*(volatile uint32_t*)0x60000014)
+#define REG_CONF0		(*(volatile uint32_t*)0x60000020)
+#define REG_CONF1		(*(volatile uint32_t*)0x60000024)
```

Comment:
Why does this access hardware register directly as opposed to going through the uart driver and why are you not using the SDK defines?